src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.text.DecimalFormat;
+
+public class Square extends Figure {
+    private double side;
+    private DecimalFormat df = new DecimalFormat("#.00");
```

Comment:
bad naming + code duplication. extract it either to some utility class, or to a Figure superclass